snippets/csharp/System/NullReferenceException/Overview/Chain2.cs

```diff
@@ -1,64 +1,71 @@
 ﻿// <Snippet7>
 using System;
 
-public class Example
+public class Chain2Example
 {
-   public static void Main()
-   {
-      var pages = new Pages();
-      Page current = pages.CurrentPage;
-      if (current != null) {
-         String title = current.Title;
-         Console.WriteLine("Current title: '{0}'", title);
-      }
-      else {
-         Console.WriteLine("There is no page information in the cache.");
-      }
-   }
+    public static void Main()
+    {
+        var pages = new Pages();
+        Page current = pages.CurrentPage;
+        if (current != null)
```

Comment:
Something here seems off if nullable references are enabled. Either `current` should be a `Page?` or the null check isn't needed.

Reply:
I have now explicitly disabled nullable context for the purpose of the examples. Can you review my most recent commit, especially the note I added to the NRE examples in the .xml file?